etc/kayobe/environments/ci-builder/inventory/hosts

```diff
@@ -3,5 +3,9 @@
 [ofed-builder:children]
 seed
 
+# This is neccesary to run `overcloud deployment image build`
```

Comment:
have you checked this won't break any other workflows?

Reply:
I'll run the host image/container build of this branch and see how it goes.

Reply:
Turns out this results in Ark repos being enabled due to being in `overcloud` now. This is now disabled [b0cf130](https://github.com/stackhpc/stackhpc-kayobe-config/pull/1414/commits/b0cf130d15f8c1d187a3ea7b90909243e468fdef)

Host image build here https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/12315268297/job/34373114113